server/admin/models.py

```diff
@@ -1,31 +1,41 @@
 from flask_login import current_user
-from flask_admin.contrib.sqla import ModelView
+from markupsafe import Markup
 
+from ..audit import AuditModelView
+from flask_admin.contrib.sqla import ModelView
 
 class UserView(ModelView):
     can_delete = True
+    can_view_details = True
     column_list = ["google_id", "first_name", "last_name", "email", "roles"]
     form_columns = ["first_name", "last_name", "email", "roles"]
 
 
-class IncidentView(ModelView):
+class IncidentView(AuditModelView):
     column_list = [
         "summary",
         "details",
         "types",
         "updated_on",
         "occurred_on",
         "reporter",
+        "audit_log_link",
     ]
     form_columns = ["summary", "details", "types", "occurred_on"]
 
+    def _audit_log_link(view, context, model, name):
+        return Markup(
+            f'<a href="/admin/auditlog/?record_id={model.id}">View Audit Logs</a>'
```

Comment:
This needs to filter by model/table, too.

Ideally this can be be done automatically. Maybe by `AuditModelView`?